### ISSUE TITLE
Crash with NullPointerException when dragging to navigation bar using Talkback

### ISSUE THREAD

**Issue by mattrussell-sonocent** [flutter/flutter#50791]
## Steps to Reproduce

1. Run `flutter create bug`.
2. Update the files as follows:  Replace `lib/main.dart` with:

```dart
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Scaffold(
        body: TextFormField(),
      ),
    );
  }
}
```

3. Run app. Enable Talkback on an Android device, and set the Android bottom navigation bar to always show. Tap and hold on the text field until the highlight border appears, then drag it down to the home button on the bottom navigation bar.

**Actual results:** Application crashes

<details>
  <summary>Logs</summary>

```bash
D/AndroidRuntime(15418): Shutting down VM
E/AndroidRuntime(15418): FATAL EXCEPTION: main
E/AndroidRuntime(15418): Process: com.example.talkback_crash, PID: 15418
E/AndroidRuntime(15418): java.lang.NullPointerException: Attempt to read from field 'int io.flutter.view.AccessibilityBridge$SemanticsNode.platformViewId' on a null object reference
E/AndroidRuntime(15418): 	at io.flutter.view.AccessibilityBridge$SemanticsNode.access$1400(AccessibilityBridge.java:1774)
E/AndroidRuntime(15418): 	at io.flutter.view.AccessibilityBridge.onAccessibilityHoverEvent(AccessibilityBridge.java:1156)
E/AndroidRuntime(15418): 	at io.flutter.embedding.android.FlutterView.onHoverEvent(FlutterView.java:538)
E/AndroidRuntime(15418): 	at android.view.View.dispatchHoverEvent(View.java:12726)
E/AndroidRuntime(15418): 	at android.view.ViewGroup.dispatchHoverEvent(ViewGroup.java:2154)
E/AndroidRuntime(15418): 	at android.view.View.dispatchGenericMotionEvent(View.java:12634)
E/AndroidRuntime(15418): 	at android.view.ViewGroup.dispatchTransformedGenericPointerEvent(ViewGroup.java:2483)
E/AndroidRuntime(15418): 	at android.view.ViewGroup.dispatchHoverEvent(ViewGroup.java:2116)
E/AndroidRuntime(15418): 	at android.view.View.dispatchGenericMotionEvent(View.java:12634)
E/AndroidRuntime(15418): 	at android.view.ViewGroup.dispatchTransformedGenericPointerEvent(ViewGroup.java:2483)
E/AndroidRuntime(15418): 	at android.view.ViewGroup.dispatchHoverEvent(ViewGroup.java:2116)
E/AndroidRuntime(15418): 	at android.view.View.dispatchGenericMotionEvent(View.java:12634)
E/AndroidRuntime(15418): 	at android.view.ViewGroup.dispatchTransformedGenericPointerEvent(ViewGroup.java:2483)
E/AndroidRuntime(15418): 	at android.view.ViewGroup.dispatchHoverEvent(ViewGroup.java:2116)
E/AndroidRuntime(15418): 	at android.view.View.dispatchGenericMotionEvent(View.java:12634)
E/AndroidRuntime(15418): 	at android.view.ViewGroup.dispatchTransformedGenericPointerEvent(ViewGroup.java:2483)
E/AndroidRuntime(15418): 	at android.view.ViewGroup.dispatchHoverEvent(ViewGroup.java:2116)
E/AndroidRuntime(15418): 	at com.android.internal.policy.DecorView.dispatchHoverEvent(DecorView.java:3180)
E/AndroidRuntime(15418): 	at android.view.View.dispatchGenericMotionEvent(View.java:12634)
E/AndroidRuntime(15418): 	at com.android.internal.policy.DecorView.superDispatchGenericMotionEvent(DecorView.java:608)
E/AndroidRuntime(15418): 	at com.android.internal.policy.PhoneWindow.superDispatchGenericMotionEvent(PhoneWindow.java:1881)
E/AndroidRuntime(15418): 	at android.app.Activity.dispatchGenericMotionEvent(Activity.java:3420)
E/AndroidRuntime(15418): 	at com.android.internal.policy.DecorView.dispatchGenericMotionEvent(DecorView.java:576)
E/AndroidRuntime(15418): 	at android.view.View.dispatchPointerEvent(View.java:12784)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:5662)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5457)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4950)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5003)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4969)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5106)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4977)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5163)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4950)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5003)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4969)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4977)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4950)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:7719)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:7659)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:7620)
E/AndroidRuntime(15418): 	at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:7830)
E/AndroidRuntime(15418): 	at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:185)
E/AndroidRuntime(15418): 	at android.os.MessageQueue.nativePollOnce(Native Method)
E/AndroidRuntime(15418): 	at android.os.MessageQueue.next(MessageQueue.java:325)
E/AndroidRuntime(15418): 	at android.os.Looper.loop(Looper.java:142)
E/AndroidRuntime(15418): 	at android.app.ActivityThread.main(ActivityThread.java:6938)
E/AndroidRuntime(15418): 	at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(15418): 	at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
E/AndroidRuntime(15418): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)
```

```console
[✓] Flutter (Channel stable, v1.12.13+hotfix.8, on Mac OS X 10.15.2 19C57, locale en-GB)
    • Flutter version 1.12.13+hotfix.8 at /Users/matt/lib/flutter
    • Framework revision 0b8abb4724 (3 days ago), 2020-02-11 11:44:36 -0800
    • Engine revision e1e6ced81d
    • Dart version 2.7.0

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/matt/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • ANDROID_HOME = /Users/matt/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3.1, Build version 11C504
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 42.1.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[!] IntelliJ IDEA Ultimate Edition (version 2019.3.1)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[!] VS Code (version 1.41.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    ✗ Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[✓] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 10 (API 29) (emulator)

```

</details>

Seen this occur on a Galaxy S8 running Android 8.0.0, and a Nexus 7 tablet running Android 6.0.1.

**Comment by TahaTesser** [comment]
Hi @mattrussell-sonocent 
Doesn't reproduce for me on Android 10 running Realme 6.
Could this be Samsung specific? Can you reproduce on the latest `stable` or `master`?

<details> 
<summary>code sample</summary> 

```dart
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: SafeArea(
        child: Scaffold(
          body: TextFormField(),
        ),
      ),
    );
  }
}

``` 

</details>

<details> 
<summary>logs</summary> 

```bash
[   +5 ms] Stopping app 'app.apk' on RMX2001.
[   +1 ms] executing: /Users/tahatesser/Code/sdk/platform-tools/adb -s EUYTFEUSQSRGDA6D shell am force-stop
com.example.master_flutter
[ +102 ms] executing: /Users/tahatesser/Code/sdk/platform-tools/adb -s EUYTFEUSQSRGDA6D shell pm list packages
com.example.master_flutter
[  +80 ms] package:com.example.master_flutter
[   +5 ms] executing: /Users/tahatesser/Code/sdk/platform-tools/adb -s EUYTFEUSQSRGDA6D shell cat
/data/local/tmp/sky.com.example.master_flutter.sha1
[  +49 ms] 4a6852a3b7301ee1082b6f607b0762584ad7a303
[   +1 ms] Installing APK.
[   +2 ms] Installing build/app/outputs/flutter-apk/app.apk...
[        ] executing: /Users/tahatesser/Code/sdk/platform-tools/adb -s EUYTFEUSQSRGDA6D install -t -r
/Users/tahatesser/AndroidStudioProjects/master_flutter/build/app/outputs/flutter-apk/app.apk
[+8844 ms] Performing Streamed Install
                    Success
[   +1 ms] Installing build/app/outputs/flutter-apk/app.apk... (completed in 8.8s)
[   +6 ms] executing: /Users/tahatesser/Code/sdk/platform-tools/adb -s EUYTFEUSQSRGDA6D shell echo -n
dfa94af549ff38be127547b8857b0e8bc3dc7a60 > /data/local/tmp/sky.com.example.master_flutter.sha1
[  +35 ms] executing: /Users/tahatesser/Code/sdk/platform-tools/adb -s EUYTFEUSQSRGDA6D shell -x logcat -v time -t 1
[ +116 ms] --------- beginning of main
                    12-04 18:57:45.646 D/AudioPolicyService(  776): UidPolicy::onUidStateChanged() UID=1000 procState=0
[  +16 ms] executing: /Users/tahatesser/Code/sdk/platform-tools/adb -s EUYTFEUSQSRGDA6D shell am start -a
android.intent.action.RUN -f 0x20000000 --ez enable-background-compilation true --ez enable-dart-profiling true --ez
enable-checked-mode true --ez verify-entry-points true
com.example.master_flutter/com.example.master_flutter.MainActivity
[  +82 ms] Starting: Intent { act=android.intent.action.RUN flg=0x20000000 cmp=com.example.master_flutter/.MainActivity
(has extras) }
[   +1 ms] Waiting for observatory port to be available...
[+3913 ms] Observatory URL on device: http://127.0.0.1:41615/86OPMoyIPi0=/
[   +1 ms] executing: /Users/tahatesser/Code/sdk/platform-tools/adb -s EUYTFEUSQSRGDA6D forward tcp:0 tcp:41615
[  +13 ms] 60280
[        ] Forwarded host port 60280 to device port 41615 for Observatory
[   +4 ms] Caching compiled dill
[  +52 ms] Connecting to service protocol: http://127.0.0.1:60280/86OPMoyIPi0=/
[   +2 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at
http://127.0.0.1:60280/86OPMoyIPi0=/.
[ +863 ms] DDS is listening at http://127.0.0.1:60282/m8jhZ0KVw-U=/.
[ +110 ms] Successfully connected to service protocol: http://127.0.0.1:60280/86OPMoyIPi0=/
[  +47 ms] DevFS: Creating new filesystem on the device (null)
[ +181 ms] DevFS: Created new filesystem on the device
(file:///data/user/0/com.example.master_flutter/code_cache/master_flutterNOBIOS/master_flutter/)
[   +4 ms] Updating assets
[  +86 ms] Syncing files to device RMX2001...
[   +1 ms] <- reset
[        ] Compiling dart to kernel with 0 updated files
[   +2 ms] <- recompile package:master_flutter/main.dart ad39df63-98b0-49ab-a4ec-c0eab811e218
[        ] <- ad39df63-98b0-49ab-a4ec-c0eab811e218
[  +73 ms] Updating files.
[        ] DevFS: Sync finished
[   +1 ms] Syncing files to device RMX2001... (completed in 78ms)
[        ] Synced 0.0MB.
[   +1 ms] <- accept
[  +17 ms] Connected to _flutterView/0x7095499020.
[   +1 ms] Flutter run key commands.
[   +1 ms] r Hot reload. 🔥🔥🔥
[        ] R Hot restart.
[        ] h Repeat this help message.
[        ] d Detach (terminate "flutter run" but leave application running).
[        ] c Clear the screen
[        ] q Quit (terminate the application on the device).
[        ] An Observatory debugger and profiler on RMX2001 is available at: http://127.0.0.1:60282/m8jhZ0KVw-U=/
[        ] Running with unsound null safety
[        ] For more information see https://dart.dev/null-safety/unsound-null-safety
[ +230 ms] I/.master_flutte(11735): ProcessProfilingInfo new_methods=407 is saved saved_to_disk=1
resolve_classes_delay=5000
[+1148 ms] V/PhoneWindow(11735): DecorView setVisiblity: visibility = 0, Parent = android.view.ViewRootImpl@ae59916,
this = DecorView@baed5bb[MainActivity]
[  +31 ms] D/Surface (11735): Surface::connect(this=0x6ffeeac000,api=1)
[  +12 ms] D/mali_winsys(11735): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig,
egl_winsys_surface **, EGLBoolean) returns 0x3000
[        ] D/Surface (11735): Surface::setBufferCount(this=0x6ffeeac000,bufferCount=4)
[        ] I/SurfaceView(11735): 207058972 visibleChanged:true -> SurfaceHolder.Callback.surfaceCreated
[        ] D/Surface (11735): Surface::connect(this=0x6f7f0ae000,api=1)
[        ] D/mali_winsys(11735): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig,
egl_winsys_surface **, EGLBoolean) returns 0x3000
[ +703 ms] D/ViewRootImpl(11735): setSurfaceViewCreated, created:true
[        ] I/SurfaceView(11735): 207058972 surfaceChanged -- format=4 w=1080 h=2268 visibleChanged:true ->
SurfaceHolder.Callback.surfaceChanged
[   +1 ms] D/SurfaceView(11735): updateScreenMode w:1080 h:2268 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(11735): try to resquest 60Hz
[        ] D/ViewRootImpl(11735): set screen refresh mode 1 due to SurfaceView-207058972
[        ] D/SurfaceView(11735): updateScreenMode w:1080 h:2268 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(11735): try to resquest 60Hz
[   +6 ms] W/Gralloc3(11735): mapper 3.x is not supported
[   +1 ms] E/gralloc (11735): Arm Module v1.0
[        ] E/ion     (11735): ioctl c0044901 failed with code -1: Invalid argument
[        ] W/gralloc (11735): WARNING: internal format modifier bits not mutually exclusive. AFBC basic bit is always
set, so extended AFBC support bits must always be checked.
[  +55 ms] I/OpenGLRenderer(11735): Davey! duration=827ms; Flags=1, IntendedVsync=364353671860460,
Vsync=364353671860460, OldestInputEvent=9223372036854775807, NewestInputEvent=0, HandleInputStart=364353676931870,
AnimationStart=364353676981716, PerformTraversalsStart=364353676985639, DrawStart=364354433519562,
SyncQueued=364354434788409, SyncStart=364354435205716, IssueDrawCommandsStart=364354435676178,
SwapBuffers=364354490333947, FrameCompleted=364354499967562, DequeueBufferDuration=3172000, QueueBufferDuration=3091000,

[+80653 ms] V/PhoneWindow(11735): DecorView setVisiblity: visibility = 0, Parent = android.view.ViewRootImpl@ae59916,
this = DecorView@baed5bb[MainActivity]
[+27237 ms] I/SurfaceView(11735): 207058972 visibleChanged:true -> SurfaceHolder.Callback.surfaceDestroyed
[   +2 ms] D/Surface (11735): Surface::disconnect(this=0x6f7f0ae000,api=1)
[   +4 ms] D/ViewRootImpl(11735): setSurfaceViewCreated, created:false
[        ] D/Surface (11735): Surface::disconnect(this=0x6f7f0ae000,api=-1)
[        ] D/SurfaceView(11735): updateScreenMode w:0 h:0 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(11735): try to resquest 90Hz
[        ] D/ViewRootImpl(11735): set screen refresh mode 0 due to SurfaceView-207058972
[   +4 ms] D/Surface (11735): Surface::disconnect(this=0x6ffeeac000,api=1)
[  +18 ms] I/GED     (11735): ged_boost_gpu_freq, level 100, eOrigin 2, final_idx 27, oppidx_max 27, oppidx_min 0
[        ] V/PhoneWindow(11735): DecorView setVisiblity: visibility = 4, Parent = android.view.ViewRootImpl@ae59916,
this = DecorView@baed5bb[MainActivity]

```

</details>

<details> 
<summary>flutter doctor -v</summary> 

```bash
[✓] Flutter (Channel master, 1.25.0-5.0.pre.64, on macOS 11.0.1 20B29 darwin-x64, locale en-GB)
    • Flutter version 1.25.0-5.0.pre.64 at /Users/tahatesser/Code/flutter_master
    • Framework revision 4d4d017aa5 (7 hours ago), 2020-12-03 21:38:10 -0800
    • Engine revision 20caf54969
    • Dart version 2.12.0 (build 2.12.0-76.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/tahatesser/Code/sdk
    • Platform android-30, build-tools 30.0.2
    • ANDROID_HOME = /Users/tahatesser/Code/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.2)
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.2, Build version 12B45b
    • CocoaPods version 1.10.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.51.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.16.0

[✓] Connected device (4 available)
    • RMX2001 (mobile)     • EUYTFEUSQSRGDA6D          • android-arm64  • Android 10 (API 29)
    • Taha’s iPad (mobile) • 00008020-000255113EE8402E • ios            • iOS 14.2
    • macOS (desktop)      • macos                     • darwin-x64     • macOS 11.0.1 20B29 darwin-x64
    • Chrome (web)         • chrome                    • web-javascript • Google Chrome 87.0.4280.88

• No issues found!
```

</details>

Can you please provide your `flutter doctor -v` your `flutter run --verbose`
Thank you

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
